examples/client-logger/app-insights-logger/package.json

```diff
@@ -0,0 +1,122 @@
+		"directory": "packages/examples/client-logger/app-insights-logger"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "concurrently npm:build:compile npm:lint",
+		"build:compile": "npm run build:tsc",
+		"build:full": "concurrently npm:build npm:webpack",
+		"build:full:compile": "concurrently npm:build:compile npm:webpack",
+		"build:tsc": "npm run tsc",
+		"clean": "rimraf coverage dist nyc *.tsbuildinfo *.build.log",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
+		"format": "npm run prettier:fix",
+		"format-and-build": "npm run format && npm run build",
+		"format-and-compile": "npm run format && npm run build:compile",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check . --ignore-path ../../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
+		"start:test-app": "start-server-and-test start:tinylicious 7070 start:test-app:client",
+		"start:test-app:client": "webpack serve --config webpack.config.js",
+		"start:tinylicious": "tinylicious",
+		"test": "npm run test:jest",
+		"test:coverage": "npm run test:jest:coverage",
+		"test:jest": "jest --detectOpenHandles",
+		"test:jest:coverage": "jest --detectOpenHandles --coverage --ci --reporters=default --reporters=jest-junite",
+		"tsc": "tsc",
+		"webpack": "webpack --env production",
+		"webpack:dev": "webpack --env development"
+	},
+	"dependencies": {
+		"@fluid-experimental/react-inputs": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/app-insights-logger": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/common-utils": "^1.1.1",
+		"@fluidframework/container-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/container-loader": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/core-interfaces": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/counter": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/datastore": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/driver-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/fluid-static": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/map": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/protocol-definitions": "^1.1.0",
+		"@fluidframework/sequence": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/shared-object-base": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@fluidframework/tinylicious-client": ">=2.0.0-internal.4.1.0 <2.0.0-internal.4.2.0",
+		"@microsoft/applicationinsights-web": "^2.8.11",
+		"react": "^17.0.1",
+		"react-dom": "^17.0.1"
+	},
```

Comment:
Let's clean this up, some of these aren't direct dependencies of the package so shouldn't need to be here.

Same for devDependencies. `@fluid-example/collaborative-textarea`, for example, isn't used AFAICT.

Reply:
I've cleaned up the dependency section from all the unused ones I could see. Should be good to go now.